src/Service/UploadFromOpenCFP/UploadedCsvCreateSpeakerTalk.php

```diff
@@ -24,26 +25,42 @@ final class UploadedCsvCreateSpeakerTalk
     private const SPEAKER_EMAIL = 'email';
 
     private $talkTitleColumnNumber;
+
```

Comment:
I would probably not add these empty lines in this PR.

Reply:
My IDE adds them automatically and the CS fixer config is fine with it, so I didn't think it would be a problem 🤷‍♂ but I guess I can remove them